src/bank.rs

```diff
@@ -35,10 +35,11 @@ pub const MAX_ENTRY_IDS: usize = 1024 * 16;
 pub const VERIFY_BLOCK_SIZE: usize = 16;
 
 /// Reasons a transaction might be rejected.
+#[cfg_attr(feature = "cargo-clippy", allow(large-enum-variant))]
 #[derive(Debug, PartialEq, Eq)]
 pub enum BankError {
     /// Attempt to debit from `Pubkey`, but no found no record of a prior credit.
-    AccountNotFound(Pubkey),
+    AccountNotFound(Transaction),
```

Comment:
Please don't. If you need additional information from the transaction, just add that. And whatever information you add, add a test that demonstrates why it's important.

Reply:
Sure, will do that.